### ISSUE TITLE
extension can behave extremely strangely when cursor tab positions are specified incorrectly

### ISSUE THREAD

**Issue by joelatschool** [artisticat1/obsidian-latex-suite#107]
I was recently trying to write a snippet, to help with bussproofs.sty, and I forgot that cursor tab position labels start at 0 instead of 1:

```json
    {trigger: "in1", replacement: "\\UnaryInfC{$1}$2", options: "m"},
```

This was my mistake, but it resulted in extremely strange behavior from the extension: it kept performing the replacement over and over on itself, fulling the screen, no matter what key I touched. Even after deleting all of the garbled latex, it kept doing strange things, duplicating text randomly and making a mess of the note. This behavior didn't stop until I restarted Obsidian.

The extension should probably check for this and refuse to use the snippet or warn the user, because right now it seems like it gets very confused to the point it could result in data loss.

**Comment by artisticat1** [comment]
Will fix; thank you for the report!

**Comment by artisticat1** [comment]
Fixed in 1.6.11.
